upup/pkg/kutil/rollingupdate_cluster.go

```diff
@@ -118,7 +130,7 @@ func (c *RollingUpdateCluster) RollingUpdate(groups map[string]*CloudInstanceGro
 		case api.InstanceGroupRoleBastion:
 			bastionGroups[k] = group
 		default:
-			return fmt.Errorf("unknown group type for group %q", group.InstanceGroup.ObjectMeta.Name)
+			return fmt.Errorf("unknown group type for group: %q", group.InstanceGroup.ObjectMeta.Name)
```

Comment:
I don't think we need the colon here.  If you wanted to print the type - which is not a bad idea - then we would.